config/job.js

```diff
@@ -60,13 +60,26 @@ const SCHEMA_STEP_OBJECT = Joi.object()
 const SCHEMA_STEP = Joi.alternatives().try(SCHEMA_STEP_STRING, SCHEMA_STEP_OBJECT);
 const SCHEMA_STEPS = Joi.array().items(SCHEMA_STEP).min(1);
 const SCHEMA_IMAGE = Joi.string();
+const SCHEMA_EMAIL = Joi.string().email();
+const SCHEMA_EMAILS = Joi.array()
```

Comment:
For simplification, Let's make `email` an object. If they want one email, still 
```yaml
email:
     addresses: [foo@example.com]
```

if they want more settings:
```yaml
email:
   addresses: [foo@example.com, bar@example.com]
   notify_developers: true
```